src/index.jsx

```diff
@@ -6,10 +6,13 @@ import ExpenseForm from "./pages/ExpenseForm";
 import ListCategories from "./pages/ListCategories";
 import ListExpenses from "./pages/ListExpenses";
 import EditCategory from "./pages/EditCategory";
+import Header from "./components/Header";
+
 
 ReactDOM.render(
     <React.StrictMode>
-        <ExpenseForm title="Editar Despesa" isEditing={true} />
+        <Header title="Despesas"/>
+         <ExpenseForm title="Editar Despesa" isEditing={true} />
```

Comment:
Penso que está na hora de usar um lint para formatação de código.

```suggestion
        <ExpenseForm title="Editar Despesa" isEditing={true} />
```

Reply:
Seria uma boa, Francis!